Makefile

```diff
@@ -11,6 +11,10 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+# THIS FILE IS AUTOMATICALLY GENERATED. DO NOT EDIT.
+# Edit https://github.com/cert-manager/makefile-modules/blob/main/modules/repository-base/base/Makefile instead.
```

Comment:
How will contributors and maintainers know how to add project specific make targets?
Please add documentation to this file explaining (or linking to an explanation) where to put project-specific make files and how they get included into this file.

Reply:
I added some extra comments.